Various editorial improvements (#198)
cfrg/draft-irtf-cfrg-frost: draft-irtf-cfrg-frost.md

# Two-Round FROST Signing Protocol {#frost-spec}

We now present the two-round variant of the FROST threshold signature protocol for producing Schnorr signatures.
It involves signer participants and a Coordinator. Signing participants are
entities with signing key shares that participate in the threshold signing

> protocol. The Coordinator is a distinguished signer with the following responsibilities:

Reviewer: ```suggestion
protocol. The Coordinator is an entity with the following responsibilities:
```

    # Compute relation values
    l = G.ScalarBaseMult(sig_share_i)

>     r = comm_share + (challenge * lambda_i * PK_i)

Reviewer: This is done in my PR -- please review that first.

* Post quantum security. FROST, like plain Schnorr signatures, requires the hardness of the Discrete Logarithm Problem.
* Robustness. In the case of failure, FROST requires aborting the protocol.

> * Downgrade prevention. All participants in the protocol are assumed to agree on what algorithms to use.

Reviewer: This is addressed in my PR -- please review that first.

## Removing the Coordinator Role {#no-coordinator}

In some settings, it may be desirable to omit the role of the Coordinator entirely.
Doing so does not change the security implications of FROST, but instead simply
requires each participant to communicate with all other participants. We loosely
describe how to perform FROST signing among signers without this coordinator role.
We assume that every participant receives as input from an external source the
message to be signed prior to performing the protocol.

Every participant begins by performing `commit()` as is done in the setting

> where a Coordinator is used. However, instead of sending the commitment

Reviewer: This was also addressed.
